### ISSUE TITLE
Error while updating property 'fill' of a view managed by: RNSVGReact.

### ISSUE THREAD

**Issue by riccardo-nigrelli** [awesomejerry/react-native-qrcode-svg#90]
Recently I would use the react-native-qrcode-svg package but I have this error:
Error while updating property 'fill' of a view managed by: RNSVGReact.
I just try the correctly working with the example that I find on the README of the corresponding repository.

I use the latest version of all package.
<code>"react-native-svg": "^11.0.1",</code>
<code>"react-native-qrcode-svg": "^6.0.3",</code>
<code>"expo": "^36.0.0",</code>

**Comment by antonleviathan** [comment]
I'm experiencing a similar issue. When I ran the app on the Android Simulator I got the same error @riccardo-nigrelli mentioned: 
`Error while updating property 'fill' of a view managed by: RNSVGReact.`.

When I test the app on my Note 4, the page which is rendering the QR code throws the following errors:

1. If the `<QRCode />` is inside of a `<Text>` component:
`Unexpected view type nested under text node: class abi35_0-0.com.facebook.react.uimanager.LayoutShadowNode`

2. And if the `<QRCode />` is not inside of a `<Text>` component:
`Invariant Violation: Invariant VIolation: Text string must be rendered within a <Text> component`

I've been able to consistently reproduce this error on different devices, and within new expo projects.
